packages/cli/package.json

```diff
@@ -8,7 +8,8 @@
     "access": "public"
   },
   "bin": {
-    "boost": "./bin/run"
+    "boost": "./bin/run",
+    "rimraf": "./node_modules/rimraf/bin.js"
```

Comment:
I don't think that this should be exposed to the user at all. Especially because they might have `rimraf` already installed. I don't see the issue, why are we exposing it like this?

Reply:
@NickSeagull by just adding the dependency in package.json I didn't manage to make `yarn clean` and `boost deploy -e production` work (rimraf command is required to be a global command to make those commands work as they are now). I totally agree that adding rimraf to bin is smell code. I suggest what I mentioned in Additional information or adding in the documentation that rimraf should be installed by the user before using booster. Nonetheless, yarn is about to disappear and maybe we can handle this by running `npx rimraf ...`, which probably will be able to get the package correctly